packages/website/containers/Header.tsx

```diff
@@ -46,7 +46,7 @@ export default function HeaderContainer({
         setMode={setMode}
         logo={mode == "dark" ? "/logoWhite.png" : "/logo.png"}
         nextLinkWrapper={(href, children) => (
-          <Link href={href}>{children}</Link>
+          <Link href={href.includes("bids") ? "bids" : href}>{children}</Link>
```

Comment:
The bids button was redirecting us outside the website , so this is the only workaround I could find

If i edit the Header component, I have to edit the cura.run code too, so I decided to just do this :)